app/src/main/java/org/walleth/activities/CreateTransactionActivity.kt

```diff
@@ -338,6 +348,24 @@ class CreateTransactionActivity : AppCompatActivity() {
         return false
     }
 
+    private fun storeDefaultGasPrice(whenDone: () -> Unit) {
```

Comment:
why pass whenDone as parameter when finish() is always the case?

Reply:
It did not feel appropriate to combine gas price and finishing

Reply:
I removed the parameter and I now also added a title